server/container/util/src/main/java/org/apache/james/util/retry/naming/LoggingRetryHandler.java

```diff
@@ -50,9 +50,7 @@ public LoggingRetryHandler(Class<?>[] exceptionClasses, ExceptionRetryingProxy p
     @Override
     public void postFailure(NamingException ex, int retryCount) {
         super.postFailure(ex, retryCount);
-        LOGGER.info(
-                "Retry failure: " + ex.getLocalizedMessage() + "\n Retrying in " + getRetryInterval(retryCount) / 1000 + " seconds"
-                );
+        LOGGER.info("Retry failure: {}\n Retrying in {} seconds", ex.getLocalizedMessage(), getRetryInterval(retryCount) / 1000);
```

Comment:
Again here the expection is swallowed.

Moreover \\n in log messages should be avoided.

What about:

```
LOGGER.info("Retry failure. Retrying in {} seconds", getRetryInterval(retryCount) / 1000, ex);
```

Reply:
Done